components/calendar/src/any_calendar.rs

```diff
@@ -781,7 +781,7 @@ impl AnyCalendarKind {
         } else if region == Some(region!("AF")) || region == Some(region!("IR")) {
             AnyCalendarKind::Persian
         } else if region == Some(region!("SA")) && algo == Some(CalendarAlgorithm::Hijri(None)) {
-            AnyCalendarKind::HijriSimulatedMecca
+            AnyCalendarKind::HijriUmmAlQura
```

Comment:
cc @sffc if we want to keep this around at all. UAQ is probably fine

Reply:
Ah, this is what Shane suggested in https://github.com/unicode-org/icu4x/issues/6330#issuecomment-2769860390. That's fine then.